resources/qml/PrintSetupSelector/Recommended/RecommendedQualityProfileSelector.qml

```diff
@@ -32,7 +32,7 @@ Item
             {
                 profileName: model.name
                 icon: model.icon
-
+                tooltipText: (model.description != undefined) ? model.description : ""
```

Comment:
checking against `undefined` is [bad practice](https://stackoverflow.com/questions/3390396/how-can-i-check-for-undefined-in-javascript) in javascript. The recommended pattern is `typeof model.description !== "undefined"`.

However, javascript also has `null` values, to also guard against that I like to write something similar to
```suggestion
                tooltipText: model.description ? model.description : ""
```